packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/interface.class.js

```diff
@@ -39,4 +40,16 @@ export default class Interface {
   hasVrack() {
     return this.vrack !== null;
   }
+
+  setVrack(vrack) {
+    this.vrack = vrack;
+  }
+
+  isTaskInProgress() {
+    return this.taskInProgress;
+  }
```

Comment:
```suggestion
  hasTaskInProgress() {
    return !!this.taskInProgress;
  }
```
if the aim is to know if there is a task